request-handler/src/main/java/au/org/aodn/aws/wps/lambda/WpsLambdaRequestHandler.java

```diff
@@ -31,6 +31,7 @@ public AwsApiResponse handleRequest(AwsApiRequest request, Context context) {
             //  Send caller a WPS error response
             AwsApiResponse.ResponseBuilder responseBuilder = new AwsApiResponse.ResponseBuilder();
             responseBuilder.statusCode(500);
+            responseBuilder.header("Content-Type", "application/xml");
```

Comment:
Why does this class exist isn't it doing exactly the same thing as its parent?

Reply:
Yes, doesn't look to be doing anything extra anymore.  Not related to the change I was doing but happy to remove it to clean up the code a bit.